### PR TITLE
Ragg2 python output format added

### PR DESCRIPTION
**Pull request by queenp**

### CODE REVIEW

File: libr/core/cconfig.c [hunk 1]
```diff
@@ -1515,6 +1515,14 @@ static int scr_ansicon(void *user, void *data) {
 }
 #endif
 
+static int cb_denseqr(void *user, void *data) {
+	RConfigNode *node = (RConfigNode *) data;
+        RCore *core = (RCore *) user;
```
Comment (radare) [inline libr/core/cconfig.c]: Tabs vs spaces

File: libr/core/cconfig.c [hunk 2]
```diff
@@ -2549,6 +2557,7 @@ R_API int r_core_config_init(RCore *core) {
 #endif
 	r_config_desc (cfg, "scr.fgets", "Use fgets() instead of dietline for prompt input");
 	SETCB ("scr.echo", "false", &cb_screcho, "Show rcons output in realtime to stderr and buffer");
+	SETCB ("scr.denseqr", "false", &cb_denseqr, "Print QR using dense UTF-8 characters");
```
Comment (radare) [inline libr/core/cconfig.c]: ❤️

**Comment by queenp** [comment]
Screwed up by accidentally including a stray commit from the qrcode PR when setting up the branch for this feature. The whitespace issue noted wasn't even on the head of where the qrcodes was. I've merged the branches to avoid conflicts at my end.
Given they're fairly simple features, I'm hoping that's okay.

**Comment by queenp** [comment]
Shall I just back out this pull request? I gather @guedou has tidied up the C printing also (which would be nice because it's not so tidy, presently, and I used similar code for the JavaScript thing I was going to implement).

**Pull request closed without merging**